.github/workflows/test.yml

```diff
@@ -142,6 +140,7 @@ jobs:
           cat ./*profile.out | grep -v "mode: atomic" >> coverage.txt
         if: env.GIT_DIFF
       - name: filter out DONTCOVER
+        # yamllint disable
```

Comment:
[Non Blocking][Suggestion]
Same as above for formatting correctly the embedded shell script

Reply:
The issue is line length, and I don't believe I can break up the lines that are violating the rule, which is why I did it this way.